### PR TITLE
Move to docker environment

### PR DESCRIPTION
**Pull request by arekkas**

### CODE REVIEW

**Comment by arekkas** [comment]
There is a bug on windows apparently which causes sometimes `docker-compose up --build` to fail with an error message that says something like `Could not write to pipe`. I think this is caused by assets having some special characters in build, but I'm not sure yet. If you encounter it let me know, ther's a workaround.

**Comment by arekkas** [comment]
Ah, apparently this bug happens when the docker images are already running and then you run `docker-compose up --build` again because you changed something in the docker configuration. Because apache2 is blocking some of the files this might cause the issues observed. Should be an edge case though, maybe document this somewhere.

**Comment by arekkas** [comment]
Apparently there seems to be an issue with the shared drive:

```
ls
ls: Zugriff auf __CG__TaxonomyEntityTaxonomyTerm.php nicht möglich: Datei oder Verzeichnis nicht gefunden
ls: Zugriff auf __CG__TermEntityTermEntity.php nicht möglich: Datei oder Verzeichnis nicht gefunden
ls: Zugriff auf __CG__UserEntityPermissionKey.php nicht möglich: Datei oder Verzeichnis nicht gefunden
__CG__TaxonomyEntityTaxonomyTerm.php  __CG__TermEntityTermEntity.php  __CG__UserEntityPermissionKey.php
foobar@DESKTOP-D93TUAV:/mnt/c/workspace/php/athene2/src/data/DoctrineORMModule/Proxy$ ls -la
ls: Zugriff auf __CG__TaxonomyEntityTaxonomyTerm.php nicht möglich: Datei oder Verzeichnis nicht gefunden
ls: Zugriff auf __CG__TermEntityTermEntity.php nicht möglich: Datei oder Verzeichnis nicht gefunden
ls: Zugriff auf __CG__UserEntityPermissionKey.php nicht möglich: Datei oder Verzeichnis nicht gefunden
insgesamt 8
drwxrwxrwx 2 root root 0 Apr 22 11:40 .
drwxrwxrwx 2 root root 0 Apr 22 11:31 ..
-????????? ? ?    ?    ?            ? __CG__TaxonomyEntityTaxonomyTerm.php
-????????? ? ?    ?    ?            ? __CG__TermEntityTermEntity.php
-????????? ? ?    ?    ?            ? __CG__UserEntityPermissionKey.php
```

Very weird.

**Comment by arekkas** [comment]
This is now good for 👀 - keep in mind that I did not install solr so you won't be able to create new revisions etc. Also it might have issues with the mailserver as I didn't set up anything here.

If you check this out in an existing directory please make sure to fetch the newest athene2-editor and also make sure to copy the configuration setting changes.

File: src/module/Markdown/src/Markdown/Options/ModuleOptions.php [hunk 1]
```diff
@@ -9,7 +9,9 @@
  */
 namespace Markdown\Options;
 
-class ModuleOptions
+use Zend\Stdlib\AbstractOptions;
+
+class ModuleOptions extends AbstractOptions
```
Comment (Knorrke) [inline src/module/Markdown/src/Markdown/Options/ModuleOptions.php]: What does this do?

Comment (arekkas) [inline src/module/Markdown/src/Markdown/Options/ModuleOptions.php]: This is a helper for injecting configuration options into the object. If it's missing, the class will not be able to properly ingest the configuration settings. I basically forgot to add it and was wondering for quite a while why I was unable to change the config of this module.

**Pull request merged**